source/lib.js

```diff
@@ -6,6 +6,30 @@
  * here eventually.
  */
 
+/**
+ * Adjust the alias of a binding property, adding an alias if one doesn't exist or
+ * replacing an existing alias. This mutates the property in place.
+ */
+function aliasBinding(p, ref) {
+  if (p.type === "Identifier") {
+    // Array element binding
+    // TODO: This ignores `name` and `names` properties of Identifier and
+    // hackily converts it to a container for a Ref.
+    p.children[0] = ref
+  } else if (p.type === "BindingRestElement") {
+    aliasBinding(p.binding, ref)
+  } else if (p.value?.type === "Identifier") {
+    // aliased property binding
+    p.value = ref
+    // TODO: nasty
+    p.children[5] = ref
```

Comment:
Indeed 😂 Could you add to the comment a pointer to which rule this is?

Reply:
I found a better way to handle it :)